test/test_cli.py

```diff
@@ -13,7 +13,36 @@
 WHOAMI_RESPONSE = {
     "account": "myaccount"
 }
+VAR_HELP = '''Usage:
+     conjur [global options] variable <subcommand> [options] <VARIABLE_ID> <VALUE>
 
+positional arguments:
+  {get,set}
+    get       Get the value of one or more variables
+    set       Set the value of a variable
+
+Options:
+  -h, --help  Display help list and exit
+'''
+
+VAR_GET_HELP = '''usage:  get [-h] variable_id [variable_id ...]
```

Comment:
same applies for this section. See my comment [above](use assertRegex or assertIn instead of assertEqual)

Reply:
In other words, I am suggesting you keep the wording we know will not likely change and remove the wording that will and assert that your text is `IN` the output and not `EQUALS` the output